homeassistant/scripts/check_config.py

```diff
@@ -246,7 +248,7 @@ def mock_logger_exception_bootstrap(msg, *params):
 
     # Patches to skip functions
     for sil in SILENCE:
-        PATCHES[sil] = patch(sil, return_value=mock_coro())
+        PATCHES[sil] = patch(sil, return_value=mock_cb())
```

Comment:
This won't work for async functions. Async functions need to return a coro. So you will need 2 SILENCE groups. Ones that are callbacks, ones that are not. Both need to be patched accordingly.

Reply:
Today we only have regular functions and @callbacks, no `async def` or @coroutines.

I commented the functions as callbacks, but can add a note to this effect?
 
Also working at not using `bootstrap.py` at all anymore, so can hopefully get rid of these soon.

Reply:
Ah, my bad. you're right.